storage-access.bs

```diff
@@ -129,6 +129,8 @@ A {{Document}} is in a <dfn>first-party-site context</dfn> if it is the [=active
 
 A {{Document}} is in a <dfn>third party context</dfn> if it is not in a [=first-party-site context=].
 
+<dfn>Global user settings</dfn> are user agent settings that can be modified by users.
```

Comment:
We don't want to require these settings to be global. I'd suggest removing this.